eox_tenant/settings/production.py

```diff
@@ -67,14 +73,19 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
         settings.TENANT_WISE_ALLOWED_PROXIES
     )
 
-    if DJANGO_CURRENT_SITE_MIDDLEWARE in settings.MIDDLEWARE_CLASSES and getattr(settings, 'USE_EOX_TENANT', False):
-        settings.MIDDLEWARE_CLASSES[settings.MIDDLEWARE_CLASSES.index(DJANGO_CURRENT_SITE_MIDDLEWARE)] = 'eox_tenant.middleware.CurrentSiteMiddleware'  # pylint: disable=line-too-long
+    # Validate if MIDDLEWARE_CLASSES attribute exists in the environment
+    middleware = 'MIDDLEWARE_CLASSES'
```

Comment:
Checking this change, my question is if we need to make the eox-tenant middlewares inherit from `django.utils.deprecation.MiddlewareMixin` (https://docs.djangoproject.com/en/2.2/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware). I checked on Juniper edx-platform some middlewares, and all of them inherit from that class

Reply:
@jfavellar90 the "eox_tenant.middleware.CurrentSiteMiddleware" is working on juniper and the older version, the other middlewares in EOX_TENANT_MIDDLEWARES require some changes, but I think that is better to scale this to the product team.

Reply:
CurrentSiteMiddleware already has that Mixin, I also think that we need to add for the other middlewares